entrypoint.sh

```diff
@@ -72,29 +85,47 @@ echo "$SUPER_USER_PASSWORD" > $FACTORY_DATA/root.pwd
 
 
 case "$DBMS_TYPE" in
-    "mariadb") 
+    "mariadb")
         DB_PORT="3306"
+        check_db_access
+
+        db_exist=$(mysql -h $DB_HOST -u $DB_USER -p$DB_PASS -e "show databases like 'coucou'")
+        if [ "$db_exist" == "" ]; then
+            echo "Database doesn't exist. Trying to create it..."
+            mysql -h $DB_HOST -u $DB_USER -p$DB_PASS -e "create database $DB_NAME"
+        fi
```

Comment:
```suggestion
        if [ "$db_exist" == "" ]; then
            echo "Database doesn't exist. Trying to create it..."
            mysql -h $DB_HOST -u $DB_USER -p$DB_PASS -e "create database $DB_NAME CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;"
        fi
```

have to be sure that the database is created in order to continue or break too